docs/source/dev_install.md

```diff
@@ -15,26 +15,29 @@ available in the PATH used with sudo.
 
         git clone https://github.com/ipython/ipywidgets
 
-1. Dev-install of the package (run from ipywidgets repo directory):
-
-        cd ipywidgets
-        pip install -v -e .
-
-2. Build the Jupyter Widgets package
+1. Build the Jupyter Widgets package
 
         cd jupyter-js-widgets
         npm install
         cd ..
 
-3. Install the Jupyter Widgets nbextension
+2. Install the Jupyter Widgets nbextension
 
         cd widgetsnbextension
         npm install
         npm run update:widgets
         pip install -v -e .
         cd ..
+        jupyter nbextension install --py --symlink widgetsnbextension
+        jupyter nbextension enable --py widgetsnbextension
+
+3. Dev-install of the package (run from ipywidgets repo directory):
+
+        cd ipywidgets
+        pip install -v -e .
+
```

Comment:
we muse install widgetsnbextensions dev before ipywidgets for the pip version not to be picked up unnecessarily.